comms/lighthouse/src/layersService.ts

```diff
@@ -45,69 +46,66 @@ export class LayersService {
   getLayerUsers(layerId: string): PeerInfo[] {
     const layer = this.layers[layerId];
     if (layer) this.checkLayerPeersIfNeeded(layer);
-    return layer?.users;
+    return this.peersService.getPeersInfo(layer!.users);
```

Comment:
Actually, since all the calls to this method are validating that the layer exists it shouldn't be null. I remember thinking that if that was the case, then I wanted this to fail.

But you are right that this should maybe be more explicit. I'm adding an error.

Thanks!